### DISCUSSION TITLE
Would ENFUGUE UI be a sensible addition?

### DISCUSSION THREAD

**Discussion by Alintya** [AbdBarho/stable-diffusion-webui-docker#538] [Ideas]
https://github.com/painebenjamin/app.enfugue.ai

I know it is meant to be plug&play already but I can imagine quite a few people would still prefer docker to not clutter up their pcs (given the fast-paced nature of SD/AI stuff in general) or just keep using their existing setup (including me)

**Comment by AbdBarho** [comment]
Hi, definitely sounds interesting, unfortunately maintenance is something that I have to deal with, with an effort that keeps increasing, so I am hesitant.

**Reply by Alintya** [reply]
After further research, https://github.com/vladmandic/automatic seems to be the better option under limited manpower/maintenance, as already mentioned in https://github.com/AbdBarho/stable-diffusion-webui-docker/discussions/407#discussion-5063794
